lib/modules/platform/github/index.ts

```diff
@@ -319,9 +319,10 @@ export async function getRawFile(
   branchOrTag?: string,
 ): Promise<string | null> {
   const repo = repoName ?? config.repository;
+  const isSameOrg = repo?.split('/')[0] === config.repositoryOwner;
```

Comment:
```suggestion
  const isSameOrg = repo?.split('/')?.[0] === config.repositoryOwner;
```
Else this will throw if repo is undefined